packages/extension-base/src/background/handlers/Tabs.ts

```diff
@@ -30,7 +30,7 @@ function transformAccounts (accounts: SubjectInfo, anyType = false): InjectedAcc
       address,
       genesisHash,
       name,
-      type
+      type: type || undefined
```

Comment:
Does it come through as `null` somewhere? (If undefined the original is fully equivalent)